portfolio/src/main/webapp/script.js

```diff
@@ -38,4 +38,14 @@ function showProjects(){
     projectContainer.style.display = "none";
   }
 
+}
+
+async function sayHello(){
+    const response = await fetch("/hello");
+    const text = await response.text();
+
+    const greeting_container = document.getElementById('greeting-container');
+    new_text = text.replace("<h1>", "")
+    new_text = new_text.replace("</h1>","")
```

Comment:
You also might consider modifying your servlet so you don't need to remove the `<h1>` part of the string? Alternatively, you could use a `<div>` as your `greeting-container` and then put the `<h1>` from the server into that.

Reply:
I tried doing it as a div, but it would not remove the quotations, I think a better solution would be to change what the servlet returns.